templates/csi-daemonset.yaml

```diff
@@ -70,7 +70,7 @@ spec:
       volumes:
         - name: providervol
           hostPath:
-            path: "/etc/kubernetes/secrets-store-csi-providers"
+            path: {{ .Values.csi.daemonSet.providersDir }}
         - name: mountpoint-dir
           hostPath:
             path: /var/lib/kubelet/pods
```

Comment:
We should probably make this public as well to be in line with [secrets-store-csi-driver/values.yaml#L70](https://github.com/kubernetes-sigs/secrets-store-csi-driver/blob/a6f86b798ecb3e301a66153303956c869c882c0f/manifest_staging/charts/secrets-store-csi-driver/values.yaml#L70)

Reply:
:+1:

Reply:
b52d1f6